Sources/NiceComponents/Button/PrimaryButton.swift

```diff
@@ -7,15 +7,17 @@
 
 import SwiftUI
 
+public typealias LocalResizableImage = ResizableImage<EmptyView>
```

Comment:
I'm not sure `LocalResizableImage` is a great name for this as it doesn't really clarify how it's different from a `ResizableImage`, especially given that it looks like most of the button APIs reference LocalResizableImage now.

Reply:
Hmm yeah not sure what a better name would be. Any suggestions?

Reply:
The generic constraint actually throws a wrench into the usage of this component more than I thought it would. If we want to keep all of this functionality, the best route may be to split this component into two. 
`ResizableImage` and `AsyncResizableImage`

Reply:
I'm not totally clear on what the intent here is, but it seems like maybe something like `DefaultResizableImage`?

Reply:
Another possibility, maybe this is a lot of hoops to jump through for what I'd expect might be a pretty low-use feature. 

Maybe we just stick with the placeholder being a loadingView, or nothing for now?

Reply:
Yeah I think it may be best to just commit the content mode fix for now and leave the bigger refactor for later. 
Closing this PR for now. Will create a new one when doing the bigger refactor